systems/common/config/config.go

```diff
@@ -187,3 +197,14 @@ func DefaultForwardConfig() Forward {
 		Path: "/",
 	}
 }
+
+func LoadServiceHostConfig(name string) *Service {
+	svcHost :="_SERVICE_HOST"
+	svcPort :="_SERVICE_PORT"
```

Comment:
I think you should check whether the enviroment variables exist before using them.
```
if host == "" {
		logrus.Errorf("environment variable %s not set", strings.ToUpper(name+svcHost))
		return nil, errors.New("environment variable not set")
	}
```

Reply:
I used os.LookuoEnv. It will set the env if present otherwise default will be used.